tests/pillar/single_swift_references.sls

```diff
@@ -0,0 +1,69 @@
+glance:
+  server:
+    enabled: true
+    version: liberty
```

Comment:
Can you please add this test also for mitaka to verify that your config files does not have any syntax error?